### ISSUE TITLE
Build failing to fetch: RPC failed; curl 18 transfer closed with outstanding read data remaining

### ISSUE THREAD

**Issue by andrew-m-leonard** [AdoptOpenJDK/openjdk-build#2286]
https://ci.adoptopenjdk.net/view/Failing%20Builds/job/build-scripts/job/jobs/job/jdk8u/job/jdk8u-linux-ppc64le-openj9/834/console
```
 > git fetch --tags --force --progress --prune -- https://github.com/AdoptOpenJDK/openjdk-build.git  +refs/pull/*:refs/remotes/origin/pr/* +refs/heads/master:refs/remotes/origin/master +refs/heads/*:refs/remotes/origin/* # timeout=10
ERROR: Error fetching remote repo 'origin'
hudson.plugins.git.GitException: Failed to fetch from https://github.com/AdoptOpenJDK/openjdk-build.git
	at hudson.plugins.git.GitSCM.fetchFrom(GitSCM.java:996)
	at hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1237)
	at hudson.plugins.git.GitSCM.checkout(GitSCM.java:1297)
	at org.jenkinsci.plugins.workflow.steps.scm.SCMStep.checkout(SCMStep.java:125)
	at org.jenkinsci.plugins.workflow.steps.scm.SCMStep$StepExecutionImpl.run(SCMStep.java:93)
	at org.jenkinsci.plugins.workflow.steps.scm.SCMStep$StepExecutionImpl.run(SCMStep.java:80)
	at org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: hudson.plugins.git.GitException: Command "git fetch --tags --force --progress --prune -- https://github.com/AdoptOpenJDK/openjdk-build.git  +refs/pull/*:refs/remotes/origin/pr/* +refs/heads/master:refs/remotes/origin/master +refs/heads/*:refs/remotes/origin/*" returned status code 128:
stdout: 
stderr: From https://github.com/AdoptOpenJDK/openjdk-build
 - [deleted]           (none)     -> origin/pr/2283/merge
error: RPC failed; curl 18 transfer closed with outstanding read data remaining
fatal: error reading section header 'acknowledgments'

	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandIn(CliGitAPIImpl.java:2450)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandWithCredentials(CliGitAPIImpl.java:2051)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.access$500(CliGitAPIImpl.java:84)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl$1.execute(CliGitAPIImpl.java:573)
	at hudson.plugins.git.GitSCM.fetchFrom(GitSCM.java:994)
```

**Comment by andrew-m-leonard** [comment]
Lot of suggestions suggest: git config --global http.postBuffer 524288000

**Comment by M-Davies** [comment]
Potentially related to https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1553. Could be another timeout related problem (although in this case, my [test PR](https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1553) is actually valid in that it does upgrade the right git command)?

**Comment by sxa** [comment]
OK ... So this is odd, but github.com was resolving to `140.82.121.4` via DNS. I forced it to use one of it's alternates `140.82.121.3` and it resolved the problem. I undid that change (so the machine resolved it as 140.82.121.4 again) and the problem has gone away. Unclear if it reset something in the TCP/IP stack or resolver somewhere, but it's now working again.
